R/randomForest_tidiers.R

```diff
@@ -740,12 +740,18 @@ exp_balance <- function(df,
       }
       else if(is.numeric(df[[col]])) {
         # for numeric cols, filter NA rows. With NAs, ubSMOTE throws mysterious error like "invalid 'labels'; length 0 should be 1 or 2"
+        # TODO: we should probably warn if more than half of rows are filtered by a column, so that user can remove that column.
         df <- df %>% dplyr::filter(!is.na(df[[col]]) & !is.infinite(df[[col]]))
       }
     }
     if (nrow(df) == 0) { # if no rows are left, give up smote and return original df.
       df_balanced <- orig_df # TODO: we should throw error and let user know which columns with NAs to remove.
     }
+    else if (n_distinct(df[[target_col]]) != 2) {
```

Comment:
why it's not "less than 2" as mentioned below?

Reply:
changed it to less than 2